crates/router/src/types/domain/payments.rs

```diff
@@ -518,3 +518,30 @@ impl From<api_models::payments::ApplePayWalletData> for ApplePayWalletData {
         }
     }
 }
+
+impl From<api_models::payments::PayLaterData> for PayLaterData {
+    fn from(value: api_models::payments::PayLaterData) -> Self {
+        match value {
+            api_models::payments::PayLaterData::KlarnaRedirect {
+                billing_email,
+                billing_country,
+            } => Self::KlarnaRedirect {
+                billing_email,
```

Comment:
billing_email and billing_country in domain model will be removed in the separate PR?

Reply:
Yes